src/components/Footer/Footer.js

```diff
@@ -0,0 +1,56 @@
+import './Footer.css';
+import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
+import messages from '../../assets/local/en';
+import { contacts } from '../../assets/contacts';
+
+const local = messages.footer;
+
+function Footer() {
+  return (
+    <footer className="footer">
+      <div className="footer__info">
+        <div className="footer__contacts">
```

Comment:
I think it would probably be best if we modified this to make it more configurable.  As it is you are limited to changing the values of a predefined set of keys: adress, phone, and a couple of others. But perhaps they'll wind up with multiple phone numbers, one for new customers, one for service requests, one to notify about robot uprisings. It would be better if they could just add a new entry

```
{
   ...
   "phoneNumber": "...",
   "serviceNumber": "...",
   ...
}
```

And this new item should be rendered like the rest. You can check out my benefits branch for an example of how to do this. Or Jessica's carousel branch, but that is actually a bit more complicated, with a nested loop.